content/pages/donations.md

```diff
@@ -7,50 +7,13 @@ author: Michael Cook
 layout: page
 url: /donations
 ---
-Project Gutenberg's success is due to the hard work of thousands of volunteers over more than 40 years. Your donations make it possible to support these volunteers, and pay our few employees to continue the creation of free electronic texts. We accept credit cards, checks and transfers from any country, in any currency.
-
-Send a **Check** or **Money Order** (any currency) to:
-
-<address>
-  Project Gutenberg Literary Archive Foundation
-</address>
-
-<address>
-  809 North 1500 West
-</address>
-
-<address>
-  Salt Lake City, UT 84116
-</address>
-
-<address>
-  United States
-</address>
-
-### Credit Card Donations
-
-Donate with a **Credit Card** by using one of these online services:
-
-<p style="text-align: center;">
-  <a title="PayPal" href="http://www.paypal.com/xclick/business=donate%40gutenberg.org&item_name=Donate+to+Gutenberg"><img title="PayPal Donate Image" src="/images/PayPal.gif" alt="PayPal Donate Image" width="62" height="31" border="0" hspace="6" /><br /> {click me}</a>
-</p>
-
-PayPal is a Web site that accepts credit card and bank transfer payments. You don't need to set up a PayPal account to donate via PayPal. Visa, MasterCard, Amex and Discover are accepted.
-
-<p style="text-align: center;">
-  <a title="NetworkForGood" href="http://www.guidestar.org/partners/networkforgood/donate.jsp?ein=64-6221541" target="_blank"><img title="Network For Good  Donate image" src="/images/Network_for_Good.gif" alt="Network For Good  Donate image" width="140" height="53" border="0" hspace="6" /><br /> {click me}</a>
-</p>
-
-This is a Web site that accepts credit card donations and charges 3% overhead (similar to most credit cards). Donation is simple, similar to most online purchase transactions. A credit card with a mailing address in the US is required.
-
-_**NOTES:**_
-
-All donations are made to the **Project Gutenberg Literary Archive Foundation** (PGLAF), which is an approved charitable 501(c)(3) organization by the U.S. Internal Revenue Service, and has the Federal Employee Information Number (EIN) 64-6221541.
+Project Gutenberg's success is due to the hard work of thousands of volunteers over more than 50 years. Your donations make it possible to support these volunteers, and pay our few employees to continue the creation of free electronic texts.
```

Comment:
> and pay our few employees

I think `our` could give the wrong impression, maybe: `and pay their few employees`?